.github/workflows/dispatch-new-version-event.yml

```diff
@@ -0,0 +1,19 @@
+name: Dispatch new version event
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+
+jobs:
+  update-webex-components:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch initiating repository event
+        # TODO: change urls for repositories
```

Comment:
@lalli-flores if you can approve despite the urls not being updated, we'll update the urls tomorrow and merge